src/main/java/Nehsik.java

```diff
@@ -0,0 +1,109 @@
+import java.util.Scanner;
+public class Nehsik {
+    public static final int MAX_TASKS = 100;
+
+    public static void main(String[] args) {
```

Comment:
I agree with the other comment, though your main function is already clear with the way you handled inputs and outputs, it could be further placed in more methods to make the main function cleaner